test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" {
+        Get-Verb | Should not be $null
+    }
+
+    It "Should get a specific verb" {
+        (Get-Verb -Verb Add | Measure-Object).Count | Should be 1
```

Comment:
>(Get-Verb -Verb Add | Measure-Obj [](start = 8, length = 33)

this should be expressed as:
`@(Get-Verb -Verb Add).Count | should be 1`
no need to use measure-object (and the same for all of these uses of Measure-Object)